build-system/tasks/helpers.js

```diff
@@ -135,6 +152,31 @@ async function compileBentoRuntime(options) {
   });
 }
 
+/**
+ *
+ * @param {*} options
```

Comment:
Strongly suggest creating a typedef for options params. One mistake we made in early AMP build code was to use an untyped options param and it grew over time, was hard to track down where/if certain props were set/used, etc.

Reply:
i absolutely hate the vague untyped `options` params everywhere too. part of why i left it `*` is that I genuinely don't know what options are expected here - I'm basing this function off the very similar compileBentoRuntime() function which is similarly barely typed. 

I can make it a little better by gathering all the options I've found used and creating a typedef for it, but i'm sure it'll  be missing some

Reply:
updated